vaadin-spring-tests/test-spring-security-flow/src/test/java/com/vaadin/flow/spring/flowsecurity/AppViewIT.java

```diff
@@ -139,6 +146,16 @@ public void private_page_logout_should_redirect_to_root() {
         assertRootPageShown();
     }
 
+    @Test
+    public void logout_redirects_to_root_page() {
+        open("login");
+        loginUser();
+        navigateTo("private");
+        assertPrivatePageShown(USER_FULLNAME);
+        clickLogout();
+        assertRootPageShown();
+    }
+
```

Comment:
This was discussed already in the Flow PR and concluded we show 404 with an additional help message in dev mode. At least for now

Reply:
So we have a different message as the result of running in dev-mode which is not covered in any tests.

Reply:
Isn't that tested in Flow in https://github.com/vaadin/flow/blob/master/flow-server/src/test/java/com/vaadin/flow/server/auth/ViewAccessCheckerTest.java#L315 ?

Reply:
It is not the same scenario, but it is not important, and it is ok since the development message is covered there. Lots of tests added, and it is hard to remember which one was testing which things! ;)